src/pages/InterviewSignupPage.tsx

```diff
@@ -25,13 +28,15 @@ const InterviewSignupPage: React.FC = () => {
   return (
     <Container maxW="container.lg" py={8}>
       <Box borderRadius="lg" boxShadow="md" p={6}>
-        <InterviewSignupForm
-          title="Select Your Availability for the following week..."
-          availability={availability}
-          onChange={handleAvailabilityChange}
-          // dayLabels={dayLabels}
-          // timeLabels={timeLabels}
-        />
+        {availability ? (
+          <InterviewSignupForm
+            title="Select Your Availability for the following week..."
+            availability={availability}
+            onChange={handleAvailabilityChange}
+          />
+        ) : (
+          <Box>Loading...</Box>
```

Comment:
Do we want to use spinners or loading text? We've been kinda using both

Reply:
I was worried spinners everywhere would be too much, but I can change to spinner

Reply:
I guess it's up to you, I like spinners personally

Reply:
No more loading text ever